### PR TITLE
Feat delete ratings comments

### PR DESCRIPTION
**Pull request by mixcas**
When a committee user is deleted all ratings related to that user are deleted and the applications affected get avg. rating recalculated.

When an admin or committee user is deleted all comments related to that user are deleted too.

### CODE REVIEW

**Comment by mixcas** [comment]
BUMP

File: app/imports/api/methods/commentsMethods.js [hunk 1]
```diff
@@ -71,3 +71,24 @@ export const deleteComment = new ValidatedMethod({
...
+  let userComments = Comments.find({
+    userId
+  }, {
+    fields: {
+      applicationId: 1
+    },
+  }).fetch();
+
+  // Get all application IDs related to the comments by the given user
+  let ratedApplicationsIds = _.map(userComments, comment => comment.applicationId);
```
Comment (varffs) [inline app/imports/api/methods/commentsMethods.js]: why do you do this here? These are applications with comments not ratings. And you don't use this variable later in the comment?

**Review by varffs — CHANGES_REQUESTED** [review]
Surplus code needs cleaning I think

**Comment by mixcas** [comment]
Ready

**Review by varffs — APPROVED** [review]

**Pull request merged**